PWGHF/TableProducer/candidateCreatorToXiPi.cxx

```diff
@@ -91,7 +91,7 @@ struct HfCandidateCreatorToXiPi {
   using MySkimIdx = soa::Filtered<HfCascLf2Prongs>;
 
   Filter filterSelectCollisions = (aod::hf_sel_collision::whyRejectColl == 0); // filter to use only HF selected collisions
-  Filter filterSelectIndexes = (aod::hf_track_index::hfflag == 1);
+  Filter filterSelectIndexes = (aod::hf_track_index::hfflag == (uint8_t)1);
```

Comment:
Do not use C-style cast. Use `static_cast` instead. (I don't know why cpplint doesn't catch this in the filter expression.)
```suggestion
  Filter filterSelectIndexes = (aod::hf_track_index::hfflag == static_cast<uint8_t>(1));
```